pytest_rts/tests/evaluation/start.py

```diff
@@ -2,83 +2,72 @@
 import logging
 import os
 import subprocess
+from pytest_rts.utils.plugin import DB_FILE_NAME
 from pytest_rts.utils.git import (
-    changed_files_between_commits,
-    file_diff_data_between_commits,
     get_current_head_hash,
-    get_test_lines_and_update_lines,
-)
-from pytest_rts.utils.db import (
-    DatabaseHelper,
-    DB_FILE_NAME,
 )
 from pytest_rts.tests.evaluation.results_db import (
     ResultDatabase,
     RESULTS_DB_FILE_NAME,
 )
-from pytest_rts.tests.evaluation.eval_helpers import (
+from pytest_rts.tests.evaluation.evaluation_utils import (
     capture_all_exit_code,
     capture_specific_exit_code,
     delete_random_line,
+    get_all_tests_for_srcfile,
+    get_file_level_tests_between_commits,
     full_diff_between_commits,
+    get_mapping_init_hash,
+    get_mapping_srcfiles,
+    get_srcfile_id,
+    get_test_suite_size,
     print_remove_test_output,
     select_random_file,
 )
-from pytest_rts.tests.testhelper import (
-    TestHelper,
-)
+from pytest_rts.tests.testhelper import TestHelper
+
+
+EVALUATION_BRANCH = "evaluation-branch"
+MAIN_BRANCH = "master"
 
 
 def random_remove_test(iterations, deletes_per_iteration, max_wait, logger):
     """Delete random lines and evaluate tests sets and pytest exitcodes"""
     if not os.path.isfile(DB_FILE_NAME):
-        logger.info("Running mapping database initialization...")
         subprocess.run(["pytest", "--rts"], check=False)
 
     results_db = ResultDatabase()
     results_db.init_conn()
     results_db.init_results_db()
 
-    mapping_db = DatabaseHelper()
-    mapping_db.init_conn()
-
-    test_suite_size = mapping_db.get_test_suite_size()
+    test_suite_size = get_test_suite_size()
     project_name = os.getcwd()
-    init_hash = mapping_db.get_last_update_hash()
-    db_size = os.path.getsize("./mapping.db")
+    init_hash = get_mapping_init_hash()
+    db_size = os.path.getsize(DB_FILE_NAME)
     project_id = results_db.store_results_project(
         project_name, init_hash, test_suite_size, db_size
     )
 
-    _, src_files = mapping_db.get_testfiles_and_srcfiles()
+    src_files = get_mapping_srcfiles()
 
     testhelper = TestHelper()
 
     for i in range(iterations):
 
         # Remove random lines
-        testhelper.checkout_new_branch()
+        testhelper.checkout_new_branch(EVALUATION_BRANCH)
         for j in range(deletes_per_iteration):
-            random_file = select_random_file(src_files)
-            filename = random_file[1]
-            delete_random_line(filename)
-            testhelper.commit_change(filename, str(j + 1))
+            random_filepath = select_random_file(src_files)
+            delete_random_line(random_filepath)
```

Comment:
Let's replace the line with `pass` statement (keeping leading whitespace) instead of deleting it. It'll give better chance of not breaking the syntax. (Comment for the future, not asking for changes in this PR)